relentless/optimize/method.py

```diff
@@ -280,57 +323,105 @@ def max_iter(self, value):
 class SteepestDescent(Optimizer):
     r"""Steepest descent algorithm.
 
-    For an :class:`~relentless.optimize.ObjectiveFunction` :math:`f\left(\mathbf{x}\right)`,
+    For an :class:`~relentless.optimize.objective.ObjectiveFunction` :math:`f\left(\mathbf{x}\right)`,
     the steepest descent algorithm seeks to approach a minimum of the function.
 
     With an initial numerical guess for all the design variables :math:`\mathbf{x}`,
     the following iterative calculation is performed:
 
     .. math::
 
-        \mathbf{x}_{n+1} = \mathbf{x}_{n}-\alpha\nabla f\left(\mathbf{x}\right)
+        \nabla f\left(\mathbf{\hat{x}}\right) = \left[{b_1}^2 \frac{\partial f}{\partial x_1},
+                                                      \cdots,
+                                                      {b_n}^2 \frac{\partial f}{\partial x_n}\right]
 
-    where :math:`\alpha` is defined as the step size hyperparameter. The step size
-    must be specified numerically, and a :class:`LineSearch` can be performed to find
+        \mathbf{x}_{n+1} = \mathbf{x}_{n}-\alpha\nabla f\left(\mathbf{\hat{x}}\right)
+
+    :math:`\mathbf{b}=\left[b_1,\ldots,b_n\right]` is defined as the scaling
+    parameters for the gradient such that :math:`\hat{x}_i=\frac{x_i}{b_i}`. By default,
+    the gradient is left unscaled (:math:`b_i=1`).
+
+    :math:`\alpha` is defined as the step size hyperparameter. The step size must
+    be specified numerically, and a :class:`LineSearch` can be performed to find
     an optimal step size value if desired.
 
+    Note that this update is equivalent to:
+
+    .. math::
+
+        \mathbf{x}_{n+1} = \mathbf{x}_{n}-c\frac{\nabla f\left(\mathbf{\hat{x}}\right)}
```

Comment:
I think Zach might have mentioned this before, but I think I would absorb the normalization of the gradient into `c`. It's cleaner to read that the direction of descent is down the gradient, and it also makes the `FixedStepDescent` clearer since it is the only class that will normalize by the gradient magnitude.